src/core/Form/StatusText/StatusText.tsx

```diff
@@ -24,10 +24,17 @@ export interface StatusTextProps extends HtmlSpanProps {
 }
 
 const StyledStatusText = styled(
-  ({ className, children, status, ...passProps }: StatusTextProps) => (
+  ({
+    className,
+    children,
+    disabled,
+    status,
+    ...passProps
+  }: StatusTextProps) => (
     <HtmlSpan
```

Comment:
I would check here if the disabled prop is true and remove aria-live if not needed.

Reply:
This should now work as described.

Reply:
Reconsidering this to use `aria-live="off"` only when set with props or when component is disabled. Otherwise it should be always on, even when there is no content.

Reply:
That might be a better option. Not sure how differently the approaches would work in practice, but this one should be a cleaner solution at least 👍

Reply:
It should now be 'off' when the element is disabled and on with the given mode in other cases - even when empty.